atom/browser/api/atom_api_session.cc

```diff
@@ -447,6 +518,36 @@ void Session::ClearHistory(mate::Arguments* args) {
                                         &task_tracker_);
 }
 
+void Session::GetHistory(mate::Arguments* args) {
+
+  base::Closure callback;
+  if (!args->GetNext(&callback))
```

Comment:
would it ever make sense to call this without a callback?

Reply:
I think so. There are/were functions that wanted to trigger a history refresh but don't actually need to wait around for the results.